python/ray/util/io_worker_func.py

```diff
@@ -0,0 +1,9 @@
+import pathlib
+from typing import List
+import logging
+logger = logging.getLogger(__name__)
+
```

Comment:
Can you move this file into _private? Util is a public module.

Also, would be great to add some comments.